src/API/PullRequestInfo.php

```diff
@@ -5,7 +5,7 @@
 /**
  * PullRequestInfo caches info about pull requests
  */
-interface PullRequestInfo
```

Comment:
Hm, php 7.3 let me get away with this?

Reply:
When I try it in 7.2.15 I get:

`PHP Fatal error:  Interfaces may not include member variables in $HOME/.terminus/plugins/terminus-build-tools-plugin/src/API/PullRequestInfo.php on line 10`

Reply:
I see -- this causes a runtime error and not a compilation error. What happens if you try and run https://gist.github.com/rvtraveller/5a9f8baec5ef7b88edca8d0d21a706dc on 7.3?

Reply:
It's not super important -- interface is clearly wrong. I'll try later. It's also possible that I failed to commit the fix, or perhaps it is fixed only in #182